src/common/services/craftApi.js

```diff
@@ -23,21 +23,16 @@ module.exports = {
           craftType,
           craftBase,
         }),
-      }, (error, response, body) => {
+      }, (error, _response, body) => {
         if (error) {
           logger.error('Failed to call craft creation API');
           reject(error);
-          return error;
```

Comment:
Promises seemed to have previously returned the error or the body, which is not really expected. Reject or resolve calls are, so these have changed.